### ISSUE TITLE
Need to wrap column values in the Table format of subscription list-filters

### ISSUE THREAD

**Issue by FarmerMike252** [pywbem/pywbemtools#1052]
This may also be true on other table format commands in the subscription manager. I've only done this on list-filters. The main problem is the query is not wrapped.

**Comment by FarmerMike252** [comment]
Here is an example of the table formatting (of just one row):
Indication Filters: submgr-id=defaultpywbemcliSubMgr, svr-id=http://FakedUrl:5988 type=all
+-------------+-------------------------------------------+-------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------+---------------------+----------------+
| Ownership   | identity                                  | Name                          | Query                                                                                                                                                                                                          | Query      | Source              |   Subscription |
|             |                                           |                               |                                                                                                                                                                                                                | Language   | Namespaces          |          Count |
|-------------+-------------------------------------------+-------------------------------+----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+------------+---------------------+----------------|
| permanent   | System Creation                           | System Creation               | SELECT * FROM CIM_InstCreation WHERE SourceInstance ISA CIM_ComputerSystem                                                                                                                                     |            | i, n, t, e, r, o, p |              0 |

**Comment by KSchopmeyer** [comment]
Issue in working branch of feature.  Note that this occurs in windows but now linux.

**Comment by KSchopmeyer** [comment]
Agreed that the wrapping is not optimum. Since this is in a working branch, we can correct in that branch,
